package.json

```diff
@@ -11,19 +11,17 @@
     "start": "./entry.sh",
     "build": "npm run typescript:release && webpack",
     "build:debug": "npm run typescript:release && npm run packagejson:copy",
-    "lint": "npm run lint:coffee && npm run lint:typescript",
+    "lint": "npm run lint:typescript",
     "test": "npm run lint && npm run test-nolint",
     "test-nolint": "npm run test:build && TEST=1 mocha",
-    "test:build": "npm run typescript:test-build && npm run coffeescript:test  && npm run testitems:copy && npm run packagejson:copy",
+    "test:build": "npm run typescript:test-build && npm run testitems:copy && npm run packagejson:copy",
     "test:fast": "TEST=1 mocha --opts test/fast-mocha.opts",
     "test:debug": "npm run test:build && TEST=1 mocha --inspect-brk",
     "prettify": "balena-lint -e ts -e js --typescript --fix src/ test/ typings/",
     "typescript:test-build": "tsc --project tsconfig.json",
     "typescript:release": "tsc --project tsconfig.release.json && cp -r build/src/* build && rm -rf build/src",
-    "coffeescript:test": "coffee -m -c -o build .",
     "packagejson:copy": "cp package.json build/",
     "testitems:copy": "cp -r test/data build/test/",
-    "lint:coffee": "balena-lint test/",
     "lint:typescript": "balena-lint -e ts -e js --typescript src/ test/ typings/ build-utils/ && tsc --noEmit && tsc --noEmit --project tsconfig.js.json",
```

Comment:
I'd say you can fold this into just `lint` now since there's no longer the coffee vs typescript diff

Reply:
I've removed any mention of :typescript since we are only using typescript now.